devel/yarn/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        yarnpkg yarn 1.12.1 v
+github.setup        yarnpkg yarn 1.12.3 v
 
 revision            1
```

Comment:
Remove the `revision` line because it starts at 0 when increasing the version and that is the default value.

Reply:
Ditto.